pkg/sdn/api/validation/validation.go

```diff
@@ -40,7 +40,7 @@ func ValidateClusterNetworkUpdate(obj *sdnapi.ClusterNetwork, old *sdnapi.Cluste
 	if obj.Network != old.Network {
 		allErrs = append(allErrs, fielderrors.NewFieldInvalid("Network", obj.Network, "cannot change the cluster's network CIDR midflight."))
 	}
-	if obj.ServiceNetwork != old.ServiceNetwork {
+	if obj.ServiceNetwork != old.ServiceNetwork && old.ServiceNetwork != "" {
```

Comment:
Old (3.0.0) nodes never look at that value anyway, so changing it while they were running would have no effect. And new nodes won't start up unless ServiceNetwork is already properly set, so there's no way it could be changed while they were running.